Defines .mtr as material and sets it to display as plain text

shadguy opened the pull request:
Defines files with .mtr extension as materials and sets them up to display as plaintext.   This is an EE-specific asset type, goes hand in hand with shaders.

shadguy committed:
Defines .mtr as material and sets it to display as plain text

virusman commented on CppProperties.json:
```diff
@@ -0,0 +1,22 @@
+{
```
Looks like these two files are not related to the PR.

shadguy replied:
Thank you, fixed.  FYI, the changes to the fmod include in two files were needed to build and test on my machine, at least for now.  I'm not sure they're required for someone more experienced.

shadguy committed:
cleaned up files incorrectly comitted.  Thanks!

virusman commented on nwnexplorer/SoundView.h:
```diff
@@ -50,7 +50,7 @@
 #include "DataRef.h"
 #include "CommonView.h"
-#include <fmod.h>
+#include "include\fmod\fmod.h"
```
Can you remove the fmod include changes? I think that'd be better for a separate PR. The code compiles as is on my machine (not sure how), and I think it may be better to fix the default include path in project settings instead.

shadguy committed:
backed out fmod include changes - works for vman without these

shadguy committed:
Merge branch 'virusman:master' into improve-resource-handling-mtr

shadguy committed:
Merge branch 'improve-resource-handling-mtr' of https://github.com/shadguy/nwnexplorer into improve-resource-handling-mtr

shadguy committed:
Minor PR cleanup.

shadguy committed:
Bump version to 1.8.4

shadguy committed:
PR cleanup

shadguy commented:
Done, thanks.  I've removed that for now, didn't figure out config yet.  
Also committed:

- version bump to 1.8.4 from 1.8.3
- changed "Material" folder name to "Materials" for more consistent grammar for end users
- cleaned up some unrelated ideas that leaked in here during fmod cleanup and the above work.  

Note: Untested, doesn't compile for me without the fmod changes.

virusman commented:
Thank you!
